src/app/task/task.component.ts

```diff
@@ -61,6 +70,7 @@ export class TaskComponent {
 				title: this.task.title,
 				description: this.task.description,
 				isCompleted: this.task.isCompleted,
+				userId: this.task.userId,
 				clickType: clickTypeParam
```

Comment:
do we need this anymore?

Reply:
it's used in onSubmit to call either addTask() or EditTask()